packages/desktop/src/main/envManager.ts

```diff
@@ -0,0 +1,30 @@
+import { Menu } from "electron";
+import getMenuTemplate from "./menu";
+import Store from "electron-store";
+import { Environment } from "../util/constants";
+
+const store = new Store();
+
+/**
+ * Loads the persisted environment from storage.
+ * Defaults to Production if no value exists.
+ */
+export function loadEnvironment(): Environment {
```

Comment:
Could we use the `PersistedStorageService` instead of this? It seems like the same purpose.